deployment/modules/github/org/secrets.tf

```diff
@@ -0,0 +1,5 @@
+resource "github_actions_organization_secret" "cloudflare_api_token_pages_upload" {
+  secret_name     = "CLOUDFLARE_API_TOKEN_PAGES_UPLOAD"
+  plaintext_value = data.terraform_remote_state.api_keys_state.outputs.terraform_key_cloudflare_pages_upload
+  visibility      = "all"
```

Comment:
Do we (maybe later?) want to scope this to only the immich-app repo?

Reply:
I've mostly been pretty lax with anything but the master keys, but we can scope this if you want to reasonably easily